modules/samples-operator-overview.adoc

```diff
@@ -57,9 +57,40 @@ Certain circumstances will result in the Samples Operator bootstrapping itself a
 
 * If the Samples Operator cannot reach link:https://registry.redhat.io[registry.redhat.io] after three minutes on initial start-up after a clean installation.
 * If the Samples Operator detects it is on an IPv6 network.
-* If the Samples Operator detects it is installed on ppc64le and S390x.
+
+However, if Samples Operator also detects an OpenShift Global Proxy is configured, it bypasses those checks.
 
 [IMPORTANT]
 ====
 IPv6 installations are not currently supported by link:https://registry.redhat.io[registry.redhat.io]. The Samples Operator pulls most of the sample imagestreams and images from link:https://registry.redhat.io[registry.redhat.io].
 ====
+
+*Restricted network installation*
+
+Boostrapping as `Removed` when unable to access `registry.redhat.io`, among other things, facilitates restricted network installations when the network restriction is already in place. Not installing the samples immediately when network access is restricted allows the cluster administrator more time to decide if samples are desired before submitting alerts that sample imagestream imports are failing, which it will start doing two hours after initial installation.
```

Comment:
Need to clean this up some as well ....

If removed, we'll never submit alerts 

that fact is unclear to me with this current text.  I'll throw up an alternative in a bit.